src/lib/Components/Artist/Articles/Article.tsx

```diff
@@ -101,15 +101,16 @@ export default createFragmentContainer(
   `
 )
 
-interface RelayProps {
-  article: {
-    thumbnail_title: string | null
-    href: string | null
-    author: {
-      name: string | null
-    } | null
-    thumbnail_image: {
-      url: string | null
-    } | null
-  }
-}
+// FIXME: This is being unused in code
```

Comment:
There were _many_ instances of there being a `interface Props { ... }` at the top of the file that had the exact same settings as `interface RelayProps { ... }` at the bottom of the file, though at the top it typically lacked `| null`. Why is this? In most instances they're nearly identical so why is one being left behind?